### PR TITLE
Add content-disposition header for csv and geojson

### PR DESCRIPTION
**Pull request by jjmurre**
# This Pull request contains changes to:

Add content-disposition header for csv and geojson download.
For a more user-friendly behaviour when downloading in the browser

Filename of the download is constructed based on {dataset_id}-{table_id}-{timestamp} if available.

# In case changes new features added

- [ ] Customer friendly documentation added into `docs/`.
- [ ] Developer friendly documentation added into `DEVELOPMENT|README.md`

# In case breaking changes introduced into API

- [ ] There is customer notification plan: ....

# In case this PR reverts changes in repo

- [ ] Extra details describing reasoning: ...

### CODE REVIEW

File: src/tests/test_dynamic_api/test_views_api.py [hunk 1]
```diff
@@ -1255,6 +1255,8 @@ def test_unpaginated_list(self, format, api_client, afval_container, filled_rout
         data = decoder(response.getvalue())
         assert data == expected_data
         assert response["Content-Type"] == expected_type  # And test after reading
+        assert "attachment" in response.get("Content-Disposition")
+        assert "afvalwegingen-containers" in response.get("Content-Disposition")
```
[the remark this answers is not in the document]
Comment (jjmurre) [reply src/tests/test_dynamic_api/test_views_api.py:1259]: Check, beter om dit te mergen tot 1 assert idd.

File: src/rest_framework_dso/views.py [hunk 2]
```diff
@@ -327,6 +328,17 @@ def finalize_response(self, request, response, *args, **kwargs):
...
+            try:
+                now = datetime.now().isoformat()
+                filename = f"{self.dataset_id}-{self.table_id}-{now}"
+            except AttributeError:
+                filename = "file"
+            content_disposition = response.accepted_renderer.get_content_disposition(filename)
+
+            if content_disposition is not None:
+                response["Content-Disposition"] = content_disposition
+
```
[the remark this answers is not in the document]
Comment (vdboor) [reply src/rest_framework_dso/views.py:341]: Waarom geen `response.accepted_renderer.finalize_response(response, dataset=..., table=...)` ? Of een variatie ervan waarbij de verantwoordelijkheden op de goede plek komen.

[the remark this answers is not in the document]
Comment (jjmurre) [reply src/rest_framework_dso/views.py:341]: Yes, of course, kan via finalize_response, tx!

[the remark this answers is not in the document]
Comment (jjmurre) [reply src/rest_framework_dso/views.py]: Aangepast.

File: src/tests/test_dynamic_api/test_views_api.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lbam) [reply src/tests/test_dynamic_api/test_views_api.py]: Misschien de extensie ook in de test opnemen voor de volledigheid?

**Review by lbam — APPROVED** [review]

**Pull request merged**